src/pages/SignUp.js

```diff
@@ -0,0 +1,42 @@
+import { StatusBar } from 'expo-status-bar';
+import React from 'react';
+import { StyleSheet, Text, View} from 'react-native';
+import { render } from 'react-dom';
+
+import Logo_SignUp from '../component/Logo_SignUp.js';
+import Form_SignUp from '../component/Form_SignUp.js';
+
+export default function SignUp() {
```

Comment:
if you do `export default function () {`, it'll automatically be assigned to `SignUp`, which follows the filename